test/src/main/java/org/axonframework/test/saga/FixtureConfiguration.java

```diff
@@ -67,6 +69,20 @@
      */
     void registerResource(Object resource);
 
+    /**
+     * Registers a {@link ParameterResolverFactory} within this fixture. The given {@code parameterResolverFactory}
+     * should be added to the other parameter resolver factories introduced through {@link
```

Comment:
`should` that's a pretty weak guarantee :)

Reply:
Lol, fair enough. Changed to `will`. However, it obviously depends on the actual implementation if it "will". Guess that's why I did `should` in the first place.